Assets/Scripts/BombSelector.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class BombSelector : MonoBehaviour
+{
+    public GameObject[] bombPrefabs;
+    private List<GameObject> currentBombs = new List<GameObject>();
+    private BombIndicator bombIndicator;
+    public int maxBombs;
+
+    void Start()
+    {
+        bombIndicator = GetComponentInChildren<BombIndicator>();
+        for (int index = 0; index < maxBombs; index++)
+        {
+            PopulateNextBomb(index);
+        }
+    }
+
+    public GameObject GetNextBomb()
+    {
+        GameObject nextBomb = DequeueBomb();
+        PopulateNextBomb(2);
```

Comment:
I think we should rename this function.  It sounds like a getter, but we also add a new bomb to our queue.  Not a big deal- but maybe worth a comment

Reply:
Renamed!